app/enrichments/remove_placeholder.rb

```diff
@@ -0,0 +1,53 @@
+##
+# Removes placeholder values from a field. Applies to both straight string 
+# values and Resources with a matching `providedLabel`. Matches are case
+# insensitive.
+#
+# @example 
+#   rp = RemovePlaceholder.new('moomin')
+#   rp.enrich_value('moomin') # => nil
+#   rp.enrich_value('MooMin') # => nil
+#   rp.enrich_value('Little My') # => 'Little My'
+#
+# @example with Resource value
+#   rp = RemovePlaceholder.new('moomin')
+#   concept = DPLA::MAP::Concept.new
+#   concept.providedLabel = 'moomin'
+#   rp.enrich_value(concept) # => nil
+#
+# @see Audumbla::FieldEnrichment
+class RemovePlaceholder
+  include Audumbla::FieldEnrichment
+
+  ## 
+  # @param [String] placeholder  a string value to regard as a placeholder
+  def initialize(placeholder = 'xyz')
+    @placeholder = placeholder
+  end
+
+  ##
+  # Removes values matching the given placeholder value, either directly or
+  # with the `#providedLabel` field.
+  #
+  # @see Audumbla::FieldEnrichment#enrich_value
+  def enrich_value(value)
+    if value.is_a? String
+      return nil if is_placeholder? value
+    elsif value.respond_to?(:providedLabel)
+      return nil if value.providedLabel.find { |l| !is_placeholder?(l) }.nil?
+    end
+    return value
+  end
+
+  private
+  
+  ##
+  # @param [String] string
+  # @return [Boolean] true if the parameter matches the placeholder, 
+  #   false otherwise
+  def is_placeholder?(string)
+    string.downcase == @placeholder.downcase
```

Comment:
Do we want to always `#downcase` the placeholder? Are there any cases where we want a strict comparison based upon string equality?